src/aggregate/aggregate_exec.c

```diff
@@ -429,7 +429,7 @@ static void sendChunk_Resp2(AREQ *req, RedisModule_Reply *reply, size_t limit,
 
     // If an error occurred, or a timeout in strict mode - return a simple error
     if (ShouldReplyWithError(rp, req)) {
-      RedisModule_Reply_Error(reply, QueryError_GetError(req->qiter.err));
+      RedisModule_Reply_Error(reply, QueryError_GetUserError(req->qiter.err));
```

Comment:
Consider using `QueryError_GetDisplayableError(req->qiter.err, false)`; avoids an extra API and clear.

Reply:
From Dvir got the comment to split the previous function into two:
- `QueryError_GetUserError`
- `QueryError_GetDisplayableError`

If you are saying I should use `QueryError_GetDisplayableError` then what is the point for `QueryError_GetUserError`?